row_cache.cc

```diff
@@ -676,9 +676,9 @@ class scanning_and_populating_reader final : public flat_mutation_reader_v2::imp
     }
 
     future<flat_mutation_reader_v2_opt> read_from_primary() {
-        auto fro = do_read_from_primary();
+        auto reader = do_read_from_primary();
         if (!_secondary_in_progress) {
-            return make_ready_future<flat_mutation_reader_v2_opt>(std::move(fro));
+            return make_ready_future<flat_mutation_reader_v2_opt>(std::move(reader));
```

Comment:
codespell is supposed to work for us, not vice versa

Reply:
Isn't this clearer?

Reply:
gents, agreed to both of you. the reason of this change is not that codespell forced me to do this. but because it made me think about the name of "fro": `fro` is far from a good variable name, and not even `from` which is the expanded form of it. but `reader` is, if we really want to use a more precise name, then `maybe_reader`, but that's might be overkill. hence i picked `reader`.

Reply:
I agree, anything that confuses codespell, has a good chance to confuse readers too.